node/api/channels_api_usage.md

```diff
@@ -90,12 +92,34 @@ in the chain, and the others are metadata used for the connection itself.
   | host | string | host of the `responder`'s node| Yes if `role=initiator` | No |
   | port | integer | the port of the `responder`s node| Yes if `role=initiator` | No |
   | role | string | the role of the client - either `initiator` or `responder` | Yes | No |
-  | timeouts | object | the maximum lenght of time waiting for the other party to respond in the different states| No | No |
   | minimum_depth | integer | the minimum amount of blocks to be mined | No | No |
 
   `responder`'s port and host pair must be reachable from `initiator` network
   so unless participants are part of a LAN, they should be exposed to the
   internet as described [here](../../node/api/README.md).
+  
+  Once established, the channel follows a [predefined set of state
+  transitions](/channels/README.md#overview). The implementation protects the
+  client from edge cases when transitions take too long or never happen using
+  a set of different timers - if the event doesn't occur in the specified time
+  frame then the off-chain protocol is considered to be violated and the
+  WebSocket connection is killed. Those are optionally configurable alongside
+  with the channel establish settings. Keep in mind that those are only local
+  values for the specific participant, protecting one's own interest. The two
+  participants can have different timeout settings and still doing updates, as
+  long as no timer fires.
+
+  All timeout values are integers and represent the waiting time in
+  milliseconds.
+
+  | Name | Description | Default value |
+  | ---- | ----------- | ------------- |
```

Comment:
I am still checking the code though I am considering whether there shall be here [two further timeouts](https://github.com/aeternity/aeternity/blob/19868f3a3f388df83fce0651a9a82203bec95bbc/apps/aehttp/src/sc_ws_handler.erl#L276-L277) i.e. `timeout_awaiting_open` and `timeout_initialized`.

Reply:
Looking in the code [how the timer is chosen](https://github.com/aeternity/aeternity/blob/19868f3a3f388df83fce0651a9a82203bec95bbc/apps/aechannel/src/aesc_fsm.erl#L255-L258) and [how states are aliased](https://github.com/aeternity/aeternity/blob/19868f3a3f388df83fce0651a9a82203bec95bbc/apps/aechannel/src/aesc_fsm.erl#L262-L280), I see that:
* `timeout_initialized` may be used for refining `timeout_accept`.
* `timeout_awaiting_open` may be used for refining `timeout_idle`.

Shall these be documented, or removed from [code looking those up](https://github.com/aeternity/aeternity/blob/19868f3a3f388df83fce0651a9a82203bec95bbc/apps/aehttp/src/sc_ws_handler.erl#L276-L277)?